File: angular_workspace/pfiva-web/src/app/data-model/message/Response.ts
import { User } from "../User";

export class Response {
	private id: number;
	private value: string;
	private user: User;

	public get Id(): number {
		return this.id;
	}
	public set Id(value: number) {
		this.id = value;
	}

	public get Value(): string {
		return this.value;
	}
	public set Value(value: string) {
		this.value = value;
	}

	public get User(): User {
		return this.user;
	}
	public set User(value: User) {
		this.user = value;
	}
}

File: workspace/data-model/src/main/java/de/pfiva/data/model/message/Response.java
package de.pfiva.data.model.message;

import java.io.Serializable;

import de.pfiva.data.model.User;

public class Response implements Serializable  {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private String value;
	private User user;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getValue() {
		return value;
	}
	public void setValue(String value) {
		this.value = value;
	}
	public User getUser() {
		return user;
	}
	public void setUser(User user) {
		this.user = user;
	}
	@Override
	public String toString() {
		return "Response [id=" + id + ", value=" + value + ", user=" + user + "]";
	}
}
